djangocms_navigation/migrations/0008_menucontent_language.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.13 on 2020-06-26 05:17
+
+import cms.utils.i18n
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('djangocms_navigation', '0007_auto_20200302_0853'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='menucontent',
+            name='language',
+            field=models.CharField(db_index=True, default=cms.utils.i18n.get_default_language, max_length=15, verbose_name='language'),
```

Comment:
I don't know how I didn't see this before. The next data migration will never run because we always set a default language here. I think this should be default="". It would have been nice to use default=get_default_language_for_site but that is not possible because we can't access model.site unfortunately. 

https://eligaservices.atlassian.net/wiki/spaces/DEV/pages/54526331/Glossary+of+Terms+Terminology